### PR TITLE
Minor changes to English redaction

### PR DESCRIPTION
**Pull request by qosobrin**
I corrected some minor errors and added some demostratives to make the text easier to read. Hope you like the changes.

### CODE REVIEW

File: manual/customize/basics-scenarios.md [hunk 1]
```diff
@@ -119,25 +119,25 @@ and subsequent rules will no longer be evaluated.
...
+    is the owner of the list.
 
   - `true()` succeeds always.
 
   - As an argument, a variable `[custom_vars->`*variable name*`]` allows you
     to introduce [custom parameters](../customize/custom-parameters.md)
     in your scenario.
 
-You can also create custom condition writing Perl module:
+You can also create custom conditions writing a Perl module:
```
Comment (racke) [inline manual/customize/basics-scenarios.md]: What about "by writing a Perl module"?

**Comment by racke** [comment]
Changes look pretty good to me 👍. Thanks @qosobrin!

File: manual/customize/basics-scenarios.md (hunk 1 above)
Comment (qosobrin) [reply manual/customize/basics-scenarios.md]: Definitely. Go with that.

Comment (ikedas) [reply manual/customize/basics-scenarios.md]: @qosobrin, you may commit that change by yourself.  If you have difficulty, please tell me.

**Pull request merged**